### DISCUSSION TITLE
PDF Annotations Question

### DISCUSSION THREAD

**Discussion by melbruki** [nextcloud/all-in-one#3841] [Questions] [answered]
I am trying to enable pdf annotations in the app store but I get an error "The appinfo/database.xml file is not longer supported. Used in pdfdraw pdf " 
If this app does not work anymore , are there any other alternatives 

thanks

**Comment by szaimen** [comment, marked as answer]
Hi, you might want to wait for Nc28 which allows for simple pdf editing.
